tensorflow/lite/kernels/BUILD

```diff
@@ -1,4 +1,5 @@
 load("//tensorflow/lite:build_def.bzl", "tflite_copts")
+load("//tensorflow/lite:build_def.bzl", "tflite_linkopts")
```

Comment:
could you merge this line with the pervious one?

Reply:
done. thanks.